apps/angularlogos/src/app/app.routes.ts

```diff
@@ -0,0 +1,8 @@
+import { Routes } from '@angular/router';
+import { LogosListComponent } from './logos-list/logos-list.component';
+import { MetacheckComponent } from './metacheck/metacheck.component';
+
+export const appRoutes: Routes = [
+  { path: '', component: LogosListComponent },
+  { path: 'metacheck', component: MetacheckComponent }
```

Comment:
I would add a `wildcard` route if possible.
Now you'll just get to a blank page, maybe it would be better to redirect the user back to the home page for now?

Reply:
ah yes, didn't think about this